src/pages/Home/index.js

```diff
@@ -182,25 +182,16 @@ const Announcement = () => (
             color: 'darkblue',
             paddingRight: '1rem',
           }}
-          className={f('small', 'icon', 'icon-species')}
-          data-icon="v"
+          className={f('small', 'icon', 'icon-common', 'icon-announcement')}
         />{' '}
-        You can see the special page created to compile all the information for:{' '}
-        <Link
-          to={{
-            description: {
-              main: { key: 'proteome' },
-              proteome: { db: 'uniprot', accession: 'UP000464024' },
-            },
-          }}
-          style={{
-            color: '#003457',
-          }}
-        >
-          {' '}
-          SARS-CoV-2
-        </Link>
-        .
+        <p>
+          The Pfam website will be decomissioned on October 5th 2022. All Pfam
+          data for this, and future versions, will be available here on the
+          InterPro website. You can read more about it in our{' '}
+          <a href="http://proteinswebteam.github.io/interpro-blog/">
+            blog post
+          </a>
+        </p>
```

Comment:
Wouldn't it be better to say that from October 5th, we will enter a transition phase during which the Pfam website (https://pfam.xfam.org) will redirect here, but a legacy website (not updated) will be available until January?

Reply:
I think most of that info will be on the Pfam landing/redirecting page. TBH I  would avoid motivating users to use the legacy instance as it will be a downgraded version, so we don't want to have as many users there as currently are in Pfam.

Reply:
But happy to change the wording tho, maybe something to talk with all the group?

Reply:
I'll merge with the current text, but if y'all agree on a different text, just do a direct commit.